moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -253,4 +253,63 @@ void ChompTrajectory::fillInMinJerk()
   }
 }
 
+void ChompTrajectory::fillInFromTrajectory(moveit_msgs::MotionPlanDetailedResponse& res)
+{
+  // create a RobotTrajectory msg to obtain the trajectory from the MotionPlanDetailedResponse object
+  moveit_msgs::RobotTrajectory trajectory_msgs = res.trajectory[0];  // = new moveit_msgs::MotionPlanResponse();
```

Comment:
rename ``trajectory_msgs`` to singular ``trajectory_msg``?

Reply:
done